chromadb/api/models/Collection.py

```diff
@@ -136,14 +137,14 @@ def peek(self, limit: int = 10) -> GetResult:
         Returns:
             GetResult: A GetResult object containing the results.
         """
-        return self._client._peek(self.id, limit)
+        return self._transform_peek_response(self._client._peek(self.id, limit))
```

Comment:
Why does async collection not need this change?

Reply:
Good catch!